conn/doc.go

```diff
@@ -4,6 +4,45 @@
 
 // Package conn defines core interfaces for protocols and connections.
 //
-// Subpackages implements all the interfaces that can be used to connects
-// peripherals.
+// Subpackages implements all the supported interfaces that can be used to
+// connects peripherals.
+//
+// Concepts
+//
+// periph uses 3 layered concepts for interfacing:
+//
+//     Bus → Port → Conn
+//
+// Not every subpackage expose all 3 concepts. In fact, most packages don't.
+// For example, SPI doesn't exxpose Bus as the OSes generally only expose the
+// Port, that is, a CS line must be selected right upfront to get an handle.
```

Comment:
Suggestion `CS line`: `Chip Select (CS) line` so developers new to SPI don't get confused by the jargon.